lib/kie/service.rb

```diff
@@ -6,6 +6,9 @@ def self.call(klass, options)
       yield init(klass)
     rescue KieClient::ApiError => err
       Rails.logger.error("KieClient::ApiError #{err.message} ")
+
+      raise Exceptions::TimedOutError.new('Connection timed out') if err.code.nil?
+      raise Exceptions::NetworkError.new(err.message) if err.code.zero?
```

Comment:
Do we expect any other services that we connect to (such as RBAC?) to ever potentially have these same errors? I know we don't currently distinguish, but these seem generic when they are really meant for the Kie service, right? Maybe itt makes more sense to have `Exceptions::KieError::TimedOut` and `Exceptions::KieError::Network` as the actual errors?

Reply:
@eclarizio Similar logic for RBAC already handled in api-common. `TimedOutError` and `NetworkError` are changed from `Kie::TimedOutError` and `Kie::NetworkError`. The reason is that they can be used by all external services.

Reply:
All the errors come from HTTP calls. They can be commonly used by all eternal http based api communication.

Reply:
@hsong-rh Ah ok, so we do expect to use these in other places then? Sounds good 👍